app/models/spotlight/dor/indexer.rb

```diff
@@ -291,27 +291,27 @@ def object_level_full_text_filenames(sdb)
       def add_manuscript_number(sdb, solr_doc)
         manuscript_number = sdb.smods_rec.location.shelfLocator.try(:text)
         return if manuscript_number.blank?
-        insert_field solr_doc, 'manuscript_number', manuscript_number, :symbol
+        insert_field solr_doc, 'manuscript_number', manuscript_number, :stored_searchable # this is a _tesim field
```

Comment:
what's the reasoning behind switching them to stored searchable? is it to tokenize them?

Reply:
Yep. Without switching them to stored searchable, I was not able to successfully search on them as a search field.